Branch-SDK/src/main/java/io/branch/referral/Defines.java

```diff
@@ -223,7 +223,10 @@ public enum Jsonkey {
         DMA_EEA("dma_eea"),
         DMA_Ad_Personalization("dma_ad_personalization"),
         DMA_Ad_User_Data("dma_ad_user_data"),
-        Is_Meta_Click_Through("is_meta_ct");
+        Is_Meta_Click_Through("is_meta_ct"),
+
+        Branch_Sdk_Request_Creation_Time_Stamp("branch_sdk_request_creation_time_stamp"),
+        Branch_Sdk_Request_Uuid("branch_sdk_request_uuid");
```

Comment:
Please match these field names with iOS - `branch_sdk_request_unique_id` and `branch_sdk_request_timestamp` .